app/components/organization-profile.js

```diff
@@ -2,4 +2,21 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   classNames: ['organization-profile'],
+
+  membership: Ember.computed('organization.organizationMemberships', 'session.currentUser', function() {
+    let user = this.get('session.currentUser');
+    if (user) {
+      let memberships = this.get('organization.organizationMemberships');
+      let membership = memberships.find(this.findMembership.bind(this));
+      return membership;
+    }
+  }),
+
+  findMembership: function(item) {
+    let itemMemberId = item.belongsTo('member').id();
+    let itemOrganizationId = item.belongsTo('organization').id();
+    let currentUserId = this.get('session.currentUser.id');
+    let organizationId = this.get('organization.id');
+    return (itemMemberId === currentUserId) && (itemOrganizationId === organizationId);
+  },
```

Comment:
We now at least know what we need. As you said, good for the first pass, but next, my idea is to have some sort of credential service. When we're working on any route that deals with an organization, we can provide the current user and the organization to that service, and then the service can figure these things out. We would have it all in one place that way.